contributing_to_docs/contributing.adoc

```diff
@@ -77,31 +102,58 @@ endif::[]
+With the combination of conditionalizing content within topics with
+`ifdef/endif` and conditionalizing whole topics with the
+link:#how-it-all-comes-together[metadata `yaml` file], the `master` branch of
+this repository always contains a complete set of documentation for all three
+OpenShift products. However, when and as new versions of an OpenShift product
+are released, the `master` branch is merged down to new or existing release
+branches. Here is the general naming scheme used in the branches:
+
+* `master` - OpenShift Origin latest code; essentially, this is our *working*
+branch
 * `origin-N.N` - OpenShift Origin most recent stable release
 * `online` - OpenShift Online most recent release
 * `enterprise-N.N` - OpenShift Enterprise support releases
 
-On a nightly basis, the documentation web sites are rebuilt for each of these branches. This way the published content for each released version of an OpenShift product will remain the same while development continues on the 'master' branch. Additionally, any corrections or additions that are "cherry-picked" into the release branches will show up in the published documentation the next day.
+On a nightly basis, the documentation web sites are rebuilt for each of these
+branches. This way the published content for each released version of an
+OpenShift product will remain the same while development continues on the
+`master` branch. Additionally, any corrections or additions that are
+"cherry-picked" into the release branches will show up in the published
+documentation the next day.
 
 [NOTE]
-All OpenShift content development occurs on the `master`, or *working* branch. Therefore, when submitting your work the PR must be created against the `master` branch. Once it is reviewed that content will then migrate to the corresponding release branches.
+====
+All OpenShift content development occurs on the `master`, or *working* branch.
+Therefore, when submitting your work the PR must be created against the `master`
+branch. Once it is reviewed that content will then migrate to the corresponding
+release branches.
+====
 
 == How it all comes together
-The documentation build system reads the `_build_cfg.yml_` metadata file to construct the content from the source files and publish to the relevant product site at https://docs.openshift.com. The build system _only_ reads this file to determine which topic files to include. Therefore, all new topics that are created must be included in the `_build_cfg.yml_` metadata file in order to be processed by the build system. 
+The documentation build system reads the `&#95;build&#95;cfg.yml` metadata file
+to construct the content from the source files and publish to the relevant
+product site at https://docs.openshift.com. The build system _only_ reads this
+file to determine which topic files to include. Therefore, all new topics that
+are created must be included in the `&#95;build&#95;cfg.yml` metadata file in
```

Comment:
Used `&#95;` to render the underscores verbatim in the above `_build_cfg.yml` references.